.github/workflows/run-end-to-end.yml

```diff
@@ -276,15 +276,9 @@ jobs:
     - name: Run DEBUGGER_PROBES_STATUS scenario
       if: always() && steps.build.outcome == 'success' && contains(inputs.scenarios, '"DEBUGGER_PROBES_STATUS"')
       run: ./run.sh DEBUGGER_PROBES_STATUS
-    - name: Run DEBUGGER_METHOD_PROBES_SNAPSHOT scenario
-      if: always() && steps.build.outcome == 'success' && contains(inputs.scenarios, '"DEBUGGER_METHOD_PROBES_SNAPSHOT"')
-      run: ./run.sh DEBUGGER_METHOD_PROBES_SNAPSHOT
```

Comment:
To not break existing CI, when scenario are removed, you must add a bit of code in `run.sh` that exit the process in success, and printing a warning telling that the scenario is removed.

Reply:
(and also plan to removes them from different CI once this Pr will be merged)

Reply:
Hmmm, done this change 5bda86d83616008b105546aeec78f03f9954f4ff, is it fine?
I believe we can run `DEBUGGER_SCENARIOS` to avoid such situations?

Reply:
Created PRs here:

python: https://github.com/DataDog/dd-trace-py/pull/11634
ruby: https://github.com/DataDog/dd-trace-rb/pull/4203
dotnet, java: using `DEBUGGER_SCENARIOS` instead of listing tests explicitly

Other libraries are not supporting debugger system tests.